scss/ui/_buttons.scss

```diff
@@ -1,5 +1,5 @@
 .btn {
-  font-weight: 600;
+  font-weight: 500;
```

Comment:
use constants as bold, normal - it's more useful - not all fonts have size as 800,400.